<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>display_settings_details_widget</class>
 <widget class="QWidget" name="display_settings_details_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="landscapeGroupBox">
     <property name="title">
      <string>Landscape</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="landscapeLabel">
        <property name="text">
         <string>Level of detail</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="landscapeHorizontalLayout" stretch="1,0">
        <item>
         <widget class="QSlider" name="landscapeSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="pageStep">
           <number>3</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="landscapeValLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Low</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="charactersGroupBox">
     <property name="title">
      <string>Characters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="charactersLabel">
        <property name="text">
         <string>Level of detail</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="charactersHorizontalLayout" stretch="1,0">
        <item>
         <widget class="QSlider" name="charactersSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="pageStep">
           <number>3</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="characterValLabel">
          <property name="text">
           <string>Low</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fxGroupBox">
     <property name="title">
      <string>FX</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="fxLabel">
        <property name="text">
         <string>Level of detail</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fxHorizontalLayout" stretch="1,0">
        <item>
         <widget class="QSlider" name="fxSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="pageStep">
           <number>3</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fxValLabel">
          <property name="text">
           <string>Low</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="texturesGroupBox">
     <property name="title">
      <string>Textures</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="texturesLabel">
        <property name="text">
         <string>Level of detail</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="texturesHorizontalLayout" stretch="1,0">
        <item>
         <widget class="QSlider" name="texturesSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="pageStep">
           <number>3</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="texturesValLabel">
          <property name="text">
           <string>Low</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>224</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
